smoker.spec

```diff
@@ -0,0 +1,58 @@
+%global branch master
+%global candidate rc2
```

Comment:
I think that using M4 or something similar from makefile just to fill in few variables would be sufficient.